src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-parso.py

```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2018, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# Hook for Jedi, a static analysis tool https://pypi.org/project/jedi/
```

Comment:
Is this meant to say Jedi instead of parso? I'm guessing parso is a dependency of Jedi?

Reply:
yes, should I state Parso instead? 
Actually, it is for IPython

Reply:
Yeah state Parso. You can mention that it's a dependency of IPython if you want.

Reply:
Done